draft-ietf-scitt-architecture.md

```diff
@@ -372,7 +372,8 @@ This section describes at a high level, the three main roles and associated proc
 ## Transparency Service
 
 Transparency Services MUST feature an Append-only Log.
-The Append-only Log is the verifiable data structure that registers Signed Statements and supports the production of Receipts.
+The Append-only Log is a verifiable data structure that accumulates all Signed Statements 
```

Comment:
I wonder if the Log is allowed to accumulate anything OTHER than "Signed Statements"... for example certificates, or JWTs.

Reply:
I preferred the previous version as it incorporates generating receipts. This also changes registered to accumulates, which only adds more concepts.

Reply:
Allowing the TS to use the log for audit information, in particular intermediate certificates or JWTs, is a pretty legitimate use case, so I would say yes

Reply:
So we are saying, @ad-l we MUST add alternatives data to Signed Statements? SHOULD?

This has been discussed a few times in calls and on the mailing list, why MUST or SHOULD it be in the log and not something computed and returned by the API? I do not oppose that change, but why open the door?

And you want the certs and JWTs themselves in the log, @ad-l? I can understand signed or transparent statements if properly sealed, but this reads as using the log for generic storage. Beyond my personal opinion, I thought many in the group oppose this notion generally.

Reply:
We need to distinguish between the log and the verifiable data structure if we are doing this. A merkle based append only log is just a bunch of hashes. The signed statements are committed to it, not "in it". I strongly prefer the original wording.

Reply:
Marking as resolved: Please see the files view for the most recent changes.